src/xdocs/index.xml.vm

```diff
@@ -304,6 +304,18 @@
                 annotations. Supports CheckStyle config files.
               </td>
             </tr>
+            <tr>
+              <td>
+                <a href="https://github.com/nikitasavinov/checkstyle-action#checkstyle-github-action">
+                Checkstyle Github Actions</a>
+              </td>
+              <td>Nikita</td>
```

Comment:
I see that name is taken from github profile, but I think last name can be added to make it `Nikita Savinov`, since username is `nikitasavinov`

Reply:
Ok, I also thought it first, but changed once see her username :)